.github/workflows/release-schema.yaml

```diff
@@ -22,11 +22,36 @@ jobs:
       - name: Get the version being released
         id: get_version
         run: echo ::set-output name=version::$(cat api-repo/schemas/latest/jsonSchemaVersion.txt)
-      - name: Overwrite Stable Json Schema in Docs
+      - name: Overwrite Stable Json Schema in Docs if needed
         run: |
-          mkdir -p docs-repo/docs/modules/user-guide/attachments/jsonschemas/stable && \
-          cp -f api-repo/schemas/latest/devfile.json \
-            docs-repo/docs/modules/user-guide/attachments/jsonschemas/stable/devfile.json
+          if [ ! -f docs-repo/docs/modules/user-guide/attachments/jsonschemas/stable/jsonSchemaVersion.txt ]; then
+            # Stable version doesn't currently exist, so just copy over the schema we're releasing
+            mkdir -p docs-repo/docs/modules/user-guide/attachments/jsonschemas/stable
+            cp -f api-repo/schemas/latest/{devfile.json,jsonSchemaVersion.txt} \
+              docs-repo/docs/modules/user-guide/attachments/jsonschemas/stable/
+            exit 0
+          fi
+
+          # Parse the schema version that's being released
+          IFS='.' read -a semver <<< "${{ steps.get_version.outputs.VERSION }}"
+          MAJOR=${semver[0]}
+          MINOR=${semver[1]}
+          BUGFIX=${semver[2]}
+          
+          # Parse the version currently set to stable
+          stableVersion=`cat docs-repo/docs/modules/user-guide/attachments/jsonschemas/stable/jsonSchemaVersion.txt`
+          IFS='.' read -a stableSemVer <<< "$stableVersion"
+          stableMajor=${stableSemVer[0]}
+          stableMinor=${stableSemVer[1]}
+          stableBugfix=$(echo ${stableSemVer[2]} | awk -F '-' '{print $1}')
+
+          # Compare the two versions, only update stable if needed
+          if ((stableMajor <= MAJOR)) && ((stableMinor <= MINOR)) && ((stableBugfix <= BUGFIX)); then
```

Comment:
For the exact same number, I can see that we don't want to update the minor version in the github action. However, I wonder if we still want to update the schema to the latest on stable in that case, i.e. should this condition be
`if ((stableMajor <= MAJOR)) && ((stableMinor <= MINOR)) && ((stableBugfix < BUGFIX)); then`  (Note: leave the github version check unchanged) ?